astrobee/config/communications/dds_generic_comms/NDDS_DISCOVERY_PEERS

```diff
@@ -0,0 +1,17 @@
+;; NDDS_DISCOVERY_PEERS - Default Discovery Configuration File
+;;Multicast builtin.udpv4://239.255.0.1 ; default discovery multicast addr
+; EVERY LINE IN THIS FILE MUST END IN A COMMENT. Even blank lines.
+;
+;; Shared Memory
+;shmem://                ; All 'shmem' transport plugin(s)
+;builtin.shmem://        ; The builtin builtin 'shmem' transport plugin
+                         ;  at network address FCC0::0
+;
+;; Unicast
+;
+;; Localhost
+127.0.0.1              ; A comma can be used a separator
+;
+;; Robot to communicate with
+;

```

Comment:
I take it this is a placeholder example file that needs to be edited for actual use of the bridge. Could you add an overview comment explaining that?

And the comments that are here could be clarified, like "Robot to communicate with" could be "Add IP address of robot to communicate with here" (assuming that's what's intended).

Also, if there are some sections we don't expect to be used for the bridge (like shared memory), maybe the file could be rearranged so they appear at the end under a section header like "Advanced features (you probably don't need to edit below this point)".